docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -608,6 +612,123 @@ fields:
+    datatype: combobox
+    choices:
+      - answer
+      - complaint
+      - motion
+      - petition
+      - response
+      - affidavit
+      - appeal
+      - notice
+      - letter
+      - application
+      - form
+      - document
+    required: False
+    show if:
+      variable: interview.customize_next_steps
+      is: True
+  - "Add the missing word: 'if the decision maker grants your ...'": interview.next_steps_document_concept
+    datatype: combobox
+    choices:
+      - request
+      - motion
+      - petition
+      - application
+      - appeal
+    required: False
+    show if:
+      variable: interview.customize_next_steps
+      is: True
+      code: |
+        interview.form_type in {"starts_case", "existing_case", "appeal"}
+  - Name of organization who can provide more help: interview.next_steps_help_organization
+    show if: interview.customize_next_steps
+    required: False
+    help: |
+      For example, a legal aid program
+  - Website with more information about this topic: interview.next_steps_help_url
+    default: ${ interview.help_page_url }
+    show if: interview.customize_next_steps
+    required: False
+    validate: |
+      lambda y: (not y or is_url(y)) or validation_error("Write a valid URL, like: https://masslegalhelp.org/my_form")
+  - Add QR code with this link?: interview.generate_next_steps_qr_code
+    datatype: yesnoradio
+    js show if: |
+      val("interview.customize_next_steps") && val("interview.next_steps_help_url") && val("interview.next_steps_help_url").length > 0
+  - note: |
+      [BR]
+      **Write one instruction per line for each question below.**
+      Each line will be its own bullet in the final document.
+    show if: interview.customize_next_steps
+  - What happens next (e.g., at the hearing)?: interview.custom_next_steps_instructions["what_happens_next"]
+    datatype: area
+    show if: interview.customize_next_steps
+    required: False
+  - What can the decision maker do?: interview.custom_next_steps_instructions["what_can_decision_maker_do"]
+    datatype: area
+    show if: interview.customize_next_steps
+    required: False
+  - What happens if my request is granted?: interview.custom_next_steps_instructions["what_happens_if_i_win"]
+    datatype: area
+    show if: interview.customize_next_steps
+    required: False
+continue button field: preview_next_steps  
+---
+continue button field: preview_final_next_steps
+id: preview final next steps
+question: |
+  Here is what your next steps instructions will look like
+subquestion: |
+  ${ next_steps_documents[interview.form_type]['attachment']["final"].pdf }
+
+  If you need to make a change, click the "back" button. You can also 
+  edit the instructions later.
+---
+template: how_to_customize_next_steps_template
+subject: |
+  How do I customize next steps in the playground?
+content: |
+  In the playground:
+
+  1. Navigate to Folders | Templates
+  2. Download the ..._next_steps.docx file to your computer
```

Comment:
We know what the full name of the document will be at this point, `interview.interview_label`, right?